File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/save-training-event-participants_param.ts


import { TrainingEventStudent_Item } from './training-event-student_item';
import { ISaveTrainingEventParticipants_Param } from './isave-training-event-participants_param';

export class SaveTrainingEventParticipants_Param implements ISaveTrainingEventParticipants_Param {
  
	public TrainingEventID: number = 0;
	public Collection?: TrainingEventStudent_Item[];
  
}

File: GTTS/INL.TrainingService.Models/SaveTrainingEventParticipants_Param.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.TrainingService.Models
{
    public class SaveTrainingEventParticipants_Param : ISaveTrainingEventParticipants_Param
    {
        public long TrainingEventID { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public List<TrainingEventStudent_Item> Collection { get; set; }
    }
}
